src/testing/flow_solver.cpp

```diff
@@ -157,10 +157,6 @@ double FlowSolver<dim, nstate>::integrate_over_domain(DGBase<dim, double> &dg,co
     std::vector<dealii::types::global_dof_index> dofs_indices (fe_values_extra.dofs_per_cell);
     for (auto cell : dg.dof_handler.active_cell_iterators()) {
         
-        if (!cell->is_locally_owned()) continue;
```

Comment:
I meant the unused comment, not the is_locally_owned

Reply:
pretty sure it will break for most mpi test without that line and the fe_values_extra.reinit